deploy/Chart/templates/controller/rbac.yaml

```diff
@@ -70,6 +70,14 @@ rules:
   - gitrepositories/status
   verbs:
   - get
+- apiGroups:
+  - "*"
+  resources:
+  - customresourcedefinitions
```

Comment:
`CustomResourceDefinitions` is required now for Flux Controller to list `GitRepository` resources

Reply:
Was this a change made by Flux?

Reply:
no, I changed the controller to watch for GitRepositories so we don't crash on startup